rh-tictactoe/src/utils/utils.js

```diff
@@ -0,0 +1,61 @@
+/* const renderMoves = history => {
+  const moves = history.map((step, stepNumber) => {
+    const desc = stepNumber ? 'Go to move #' + stepNumber : 'Go to game start';
+    const li = (
+      <li key={stepNumber}>
+        <button onClick={() => this.props.jumpTo(stepNumber)}>{desc}</button>
+      </li>
+    );
+    return li;
+  });
+  return moves;
+}; */
+
+export function calculateWinner(squares) {
+  let winner = null;
```

Comment:
set it on false and then you can use array properties like `some` to make the job easier. We can check this later

Reply:
I set it to false but i have some questions about using array.some. But as you said before we can check this later.